lisp/ess-inf.el

```diff
@@ -125,6 +125,13 @@ If `ess-plain-first-buffername', then initial process is number-free."
                         (= n 1))) ; if not both first and plain-first add number
               (concat ":" (number-to-string n)))))
 
+(defvar-local inferior-ess--start-name nil
+  "Program name used to start the buffer.
+Buffer-local in inferior buffers.")
```

Comment:
No need to say that it's buffer local if you're defining it with defvar-local. It'll already have that in the docstring.